peril.settings.json

```diff
@@ -39,21 +37,5 @@
       "pull_request": "dangerfile.ts"
       // "pull_request.review_requested": "org/addReviewer.ts"
     }
-  },
-  "tasks": {
```

Comment:
These are unrelated to the Peril Bot [alerts](https://artsy.slack.com/archives/C02BC3HEJ/p1747405286822259)?

Reply:
They come from this a few lines up:

```json
"issues.labeled": ["org/rfc/scheduleRFCsForLabels.ts"],
```